content/_data/indexpage/carousel.yml

```diff
@@ -1,3 +1,13 @@
+# 2022 Elections
+- :title: Jenkins Elections 2022
+  :intro: Voting for the Jenkins Board and Officer Elections 2022 begin this Thursday, Nov 17.
+    Voter registration is happening now until Nov 17
```

Comment:
maybe "open" instead of "happening" or "possible"?

Reply:
I personally prefer "happening", as "right now" & more engaging than "possible".
Also, they're open since quite some time already, they haven't just "opened".

Reply:
makes sense! No worries on changing anything